dm/checker/check_test.go

```diff
@@ -247,131 +247,6 @@ func (s *testCheckerSuite) TestBinlogRowImageChecking(c *tc.C) {
 	c.Assert(err, tc.IsNil)
 }
 
-func (s *testCheckerSuite) TestTableSchemaChecking(c *tc.C) {
```

Comment:
this has been tested in https://github.com/pingcap/tiflow/blob/f710abba6417971370e0d0711d9cd8ca3fa784c4/dm/tests/dmctl_basic/run.sh#L284

the task file is https://github.com/pingcap/tiflow/blob/master/dm/tests/dmctl_basic/conf/dm-task3.yaml

check tables is https://github.com/pingcap/tiflow/blob/f710abba6417971370e0d0711d9cd8ca3fa784c4/dm/tests/dmctl_basic/data/db1.prepare.sql#L26-L27

the failed check items are 
1. sharding table columns are different
2. not exist PK/UK

Reply:
But why remove them? Seems they are different test(UT and IT)

Reply:
Indeed. I will revert them later